cpp/ql/src/semmle/code/cpp/Function.qll

```diff
@@ -334,6 +334,18 @@ class Function extends Declaration, ControlFlowNode, AccessHolder, @function {
     )
   }
 
+  /**
+   * Gets the class of which this function, called `name`, is a member.
```

Comment:
The parameter is called `myName`, but the QLDoc refers to `name`. Should these be the same, or am I reading the QLDoc incorrectly?

Reply:
Which do you prefer?  (since I don't seem to have been able to make my mind up)

Reply:
Or perhaps `memberName`?

Reply:
I like `memberName` 👍

Reply:
Done.